### DISCUSSION TITLE
Verification process discriminating against people with hyphenated last names

### DISCUSSION THREAD

**Discussion by sjohnsonyu** [community/community#60333] [GitHub Education]
### Hi everyone,

I am continually being rejected with the error message:

> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio. The verification process looks at your GitHub profile to cross reference the evidence submitted. Do you use a variation of your name or a nickname? Is your name in non-english? When you update your profile information please log out and log back into GitHub before re-applying.

My bio is complete, and trying variants of my name has led me to conclude that these rejections are due to my last name being hyphenated. My proof of enrollment lists my name as: LASTNAME1-LASTNAME2, FIRST MIDDLE and I have tried these variants:

- FIRST MIDDLE LASTNAME1-LASTNAME2
- FIRST LASTNAME1-LASTNAME2
- FIRST MIDDLEINITIAL LASTNAME1-LASTNAME2
- LASTNAME1-LASTNAME2, FIRST MIDDLE
- FIRST MIDDLE LASTNAME1 LASTNAME2

Please advise on how to proceed.

**Comment by samus-aran** [comment]
Hi @sjohnsonyu 

Whilst the image detection is robust at times it does misinterpret. I would recommend you use the exact format of your name on your piece of evidence as we do detect hyphens. We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍
